core/tests/puppeteer-acceptance-tests/utilities/user/logged-in-user.ts

```diff
@@ -47,6 +47,8 @@ const signUpUsernameField = 'input.e2e-test-username-input';
 const invalidEmailErrorContainer = '#mat-error-1';
 const invalidUsernameErrorContainer = '.oppia-warning-text';
 const optionText = '.mat-option-text';
+const profileButton = '.e2e-test-profile-dropdown';
+const learnerDashboardDropdown = '.e2e-test-learner-dashboard-link';
```

Comment:
@seanlip I am bit unsure, if these variable are correct or not?

Reply:
Suggest:

```
const userDropdown = '.e2e-test-user-dropdown'
const learnerDashboardMenuLink = '.e2e-test-learner-dashboard-menu-link'
```

Reply:
@seanlip Since the element is 'e2e-test-profile-dropdown' and all the variable names for this element in other files are 'Profile dropdown', maybe we should name it 'Profile dropdown' for consistency?

Reply:
Ah thanks ... yeah, if we already have a convention of "profile dropdown" then let's stick with that. Thanks for calling that out @imchristie!

Reply:
Done!